README.md

```diff
@@ -39,7 +39,7 @@ run this to build & run the storybook locally:
 ```bash
 npm run storybook
 ```
-the storybook will be served on `http://localhost:7007`
+the storybook will be served on `http://localhost:7008`
```

Comment:
Why? I don't think it's correct

Reply:
Just so it be more convenient - monday-ui-components also runs on localhost:7007 and when u r running them together u need to make more actions like 'press y, so it would run on a different port'...
Decided that it's better to change it here